soh/soh/SohGui/SohMenuBar.cpp

```diff
@@ -1528,8 +1528,6 @@ void DrawEnhancementsMenu() {
             UIWidgets::Tooltip("Restore pre-release behavior where defeating a Gold Skulltula will play a cutscene showing it die.");
             UIWidgets::PaddedEnhancementCheckbox("Quick Bongo Kill", CVAR_ENHANCEMENT("QuickBongoKill"), true, false);
             UIWidgets::Tooltip("Restore a bug from NTSC 1.0 that allows bypassing Bongo Bongo's intro cutscene to quickly kill him");
-            UIWidgets::PaddedEnhancementCheckbox("Original RBA Values", CVAR_ENHANCEMENT("RestoreRBAValues"), true, false);
-            UIWidgets::Tooltip("Restores the original outcomes when performing Reverse Bottle Adventure.");
```

Comment:
this is the biggest reason i'd like to see this land. having (R)BA "just work" is much better than having people check a box to restore the values